### ISSUE TITLE
[React Native Gallery App : Switch]: While navigating to 'Switch' control elements using keyboard, keyboard focus is not visible clearly

### ISSUE THREAD

**Issue by vineethkuttan** [microsoft/react-native-windows#15641]
## ADO Bug Reference

| Field | Value |
|-------|-------|
| **ADO Link** | 60965066 |
| **Type** | Bug |
| **State** | Active |
| **Priority** | -- |
| **Severity** | -- |
| **Reason** | Triage Needed |
| **Area Path** | `--` |
| **Iteration** | `--` |
| **Created By** | Sreevidya Vemuri (WIPRO LIMITED) |
| **Assigned To** | Nitin Chaudhary |

---

## Repro Steps

Environment Details:
OS Version: 25H2
OS Build: 26200.7634
Application: React Native Gallery
Application Version: 2.1.0.0

Repro Steps:

1. Open the React native Gallery application and sign in with valid credentials.
2. Now navigate to 'Navigation menu' button and activate it.
3. Now navigate to 'Basic input' and activate it.
4. Now navigate to 'Switch' and activate it.
5. Now navigate to 'Simple Switch/Colored switch' and observe the keyboard focus. Actual Results:

While navigating to 'Switch' control elements using keyboard, keyboard focus is not visible clearly. When Switch is focused, small circle is getting appeared on the 'Switch' control which is difficult to identify.

Expected Results:
While navigating to 'Switch' control elements using keyboard, full rectangle should appear on the Switch.

### User Impact

Users who rely on keyboard and low vision users will face difficulty in Understanding the focused element if focus is not visible.

_MAS Reference: [MAS 2.4.7 – Focus Visible](https://aka.ms/MAS2.4.7)_

_Trap ID: [9.1 - Unattractive Appearance](https://aka.ms/UnattractiveAppearance)_

"If the bug is invalid, incomplete, wrong severity, has unclear reproduction steps or attachments, please tag it with "**A11yRCA**" and provide a detailed comment on why the **A11yRCA** tag was added. However, do not add the "**A11yRCA**" tag if the bug is related to partners or if you have any questions about the bug."

---

> _Automatically created from ADO Bug 60965066 using ADO to GitHub extension_
